BlueMap/Chat/src/main/resources/bluemap-chat.css

```diff
@@ -70,3 +119,22 @@
         display: none;
     }
 }
+
+#toggle-chat-view-control {
+    aspect-ratio: 1;
+    background-color: transparent;
+}
+
+#toggle-chat-view-control:hover {
+    background-color: var(--theme-bg-hover);
+}
+
+#toggle-chat-view-control:active {
+    background-color: var(--theme-fg-light);
+    color: var(--theme-bg)
+}
```

Comment:
These opaque colors on the translucent background and default state feel really out of place, maybe just use more or less translucent background colors?

Reply:
I was trying to keep the theme of the Map. It looks alright for me in dark mode. Are you testing in light mode?

Reply:
![Screenshot_2024-08-16-09-23-16-614_org mozilla firefox~2](https://github.com/user-attachments/assets/9f4e3de6-fc30-419f-aafc-17d392b316cd)
![Screenshot_2024-08-16-09-23-21-208_org mozilla firefox~2](https://github.com/user-attachments/assets/5be623fa-2b2a-4937-a310-e5bf320b1eba)
![Screenshot_2024-08-16-09-23-24-001_org mozilla firefox~2](https://github.com/user-attachments/assets/1fdb2e5f-7176-495a-8c0e-136229a79cb3)

Just normal dark mode, see how the chat is otherwise transparent, and so is the button in the default state, but when hovered or clicked it becomes opaque

Reply:
I will try to make adjustments

Reply:
I have been trying to make some changes the the button. Its hard to make it look good and keep it consistent with the Bluemap theme. Here is what I have got so far... what do you think will be good to do? Bluemap uses CSS variables which update if light/dark. If we want to keep the chat UI consistent with the rest of Bluemap then we should use them. 

![image](https://github.com/user-attachments/assets/cf027f21-efc3-4a1a-8c17-4aadda965bd7)
![image](https://github.com/user-attachments/assets/1700d5b3-6b53-4d9c-896a-bbaf009565a9)
![image](https://github.com/user-attachments/assets/8e1248f5-3fc3-4988-85a8-4693e8af1640)

Reply:
The chat UI isn't really consistant with the rest of BlueMap anyway due to the transparency (just pure black at 0.25 opacity) and doesn't use BlueMap's dark/light variables anywhere. I think it's more important for the button to fit in with the chat UI rather than BlueMap UI. I think what you currently have is good enough 👍
Though hmm... would it look any good if only the icon changed color and not the background? Then maybe the usage of BlueMap's colors would be totally fine?

Reply:
Yeh ok. I will make a few more adjustments. I currently use bluemap colors for the icon color. However it doesn't update because the chat ui isn't in the "app" div. (If we use bluemap colors I will move the chat ui location so that it integrates more with everything. Also if we only changed the icon color then you won't be able to see the image under certain conditions.

Reply:
Changed the hover and click colors.

![image](https://github.com/user-attachments/assets/7dcf66ae-f17d-4a8f-8134-19f464ace7f9)
![image](https://github.com/user-attachments/assets/4f7c7b10-649b-4a36-8bdf-fae581320821)

Feels a bit better and closer to the ui of the chat

Reply:
:+1: